doc/source/Reference/ScriptingReference/index.md

```diff
@@ -6,3 +6,16 @@ Scripting Reference
 - [Expressions](Expressions/index.md)
 - [Set Expressions](SetExpressions/index.md)
 - [Metadata](Metadata/index.md)
+
+<!-- TOC -->
+
+```eval_rst
+.. toctree::
+    :hidden:
+
+    CommonOperations/index.md
```

Comment:
Do we have to add this TOC stuff per-file to make the TOC behave? That seems quite error prone. Is there any matching syntax so we could say "*/*.md" or something else that means "hide all child pages" instead?

Reply:
Sure, that would work. There will be a few instances where you still want static ones, though, like for separators.